app/models/job.rb

```diff
@@ -3,4 +3,8 @@ class Job < ApplicationRecord
   validates :location, presence:true
   validates :company, presence:true
   validates :link, presence:true
+  validates :status, presence:true
```

Comment:
atm you validate that the status is provided but don't do any backend validation that the status is valid. 

For example, I could do

```
job = Job.new 
job.status = "bananas"
job.save
```
and it would save. 

You have some protection in the frontend (can only select the status by dropdown) but it still is risky leaving this unprotected on the backend 

See if you can make it so that the `status` has to be one of the options here

Reply:
shall do!